.travis.yml

```diff
@@ -54,7 +54,7 @@ matrix:
       env: LIBRABBITMQ_VERSION=v0.5.2 TEST_PHP_ARGS=-m
 
 before_install:
-  - sh provision/install_rabbitmq-c.sh ${LIBRABBITMQ_VERSION}
+  - sh -c 'provision/install_rabbitmq-c.sh'
```

Comment:
Is it intentional that `LIBRABBITMQ_VERSION` removed from args?

Reply:
The argument wasn't used in the script which directly use the environment variable.

Reply:
My bad, it was using this argument. I can add it back if you prefer.